src/App.tsx

```diff
@@ -1,57 +1,22 @@
-import React from 'react';
+import React, { useState, FC } from 'react';
 import './App.scss';
-import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { Dropdown } from './components/Dropdown/Dropdown';
 
-export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+export const App: FC = () => {
+  const [selectedPerson, setSelectedPerson] = useState<Person>();
```

Comment:
better to specify that initial user is `null`
```suggestion
  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
```